tests/functional/sign_in.js

```diff
@@ -113,7 +113,7 @@ define([
       var email = TestHelpers.createEmail();
       return fillOutSignIn(this, email, PASSWORD)
         // The error area shows a link to /signup
-        .then(FunctionalHelpers.visibleByQSA('.error a[href="/signup"]'))
+        .then(FunctionalHelpers.visibleByQSAHeight('.error a[href="/signup"]', undefined, 10))
```

Comment:
can we also make this change for other "errors with links" that we have in our tests, i noticed some account unlock tests also fail when animation is slow:

see `a[href="/confirm_account_unlock"]`, `.error a[href="/signin"]`, `a[href="/signup"]`, `a[href="/oauth/signin"]`, `a[href="/oauth/signup"]` and ` in other functional tests